rsync_tmbackup.sh

```diff
@@ -42,7 +42,7 @@ fn_parse_date() {
 }
 
 fn_find_backups() {
-    fn_run_cmd "find "$DEST_FOLDER" -type d -name "????-??-??-??????" -prune | sort -r"
```

Comment:
Sorry for the very late reply. We actually need the quotes around `$DEST_DIR` since the path might contain space characters. I think a more robust code would be:

    fn_run_cmd "find \"$DEST_FOLDER\" -type d -name \"????-??-??-??????\" -prune | sort -r"

I'd be glad to merge the PR if you could update it like this.